mlibrary/catalog-browse
# Slim to erb

# Overview
It has been unanimously decided to drop Slim and roll back to using `.erb` templates. This pull request:
* Converts `./views/browse.slim` to an `.erb` file `./views/layout.erb`.
* Has `./catalog-browse.rb` point to the new `.erb` file.
* Removes the `slim` gem. 
* Removes unused files.

## Testing
- Install the gems (`docker-compose run --rm web bundle install`)
- Run the Docker contains to make sure everything still works (`docker-compose up`)
- Compare [./views/browse.slim](https://github.com/mlibrary/catalog-browse/blob/main/views/browse.slim) to `./views/layout.erb` to make sure everything has been converted correctly.
